docs/Userguide.rst

```diff
@@ -0,0 +1,114 @@
+************************************
+UI support to create and manage Kubernetes cluster for FIWARE Lab user
+************************************
+
+Fiware Lab user will be able to create and manage kubernetes cluster through UI. User can also deploy Fiware GEs over kubernetes cluster by selecting the GEs from the catalogue provided in GUI.
+
+User can redirect himself to kubernetes GUI from FIWARE Lab dashboard afte clicking on kubernetes button.
+
+After successful login to FIWARE Lab a button will appear in overview section of Project from where the user will be redirected to kubernetes GUI where user will get options to create and manage kubernetes cluster and deploy Fiware GEs.
+
+Pages in the FIWARE Lab GUI:
+---------------------------------------------
+**Login Page:** User have to login with their credentials to access FIWARE Lab dashboard.
+
+.. figure:: source/figures/login-page.png
+   :align: center
+   :width: 100px
+   :height: 100px
+   :scale: 50 %
+
+**Instances:** User have to create two instances in instances tab of FIWARE Lab dashboard with minimum configuration flavor medium(2VCPUS).
```

Comment:
change statement as:
User have to create two instances in instances tab of FIWARE Lab dashboard with configuration as follows:
Flavor: medium or large
OS: Centos

Reply:
Changes were done